invenio_rdm_records/theme/templates/invenio_rdm_records/macros/files.html

```diff
@@ -0,0 +1,39 @@
+{#
+    Copyright (C) 2020 CERN.
+    Copyright (C) 2020 Northwestern University.
+  
+    Invenio App RDM is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{%- macro file_list(files) %}
```

Comment:
There should already be a marco available - see cookiecutter-invenio-instance for where it is.

Reply:
That one was already in app-rdm from previous releases, so I just moved it here. It's in my todo list to analyze which macros exist and if some can be moved to other repos :)